neutrinordp/xrdp-neutrinordp.c

```diff
@@ -1390,9 +1380,9 @@ lfreerdp_pointer_new(rdpContext *context,
     }
     else
     {
-        LLOGLN(0, ("lfreerdp_pointer_new: error bpp %d width %d height %d index: %d",
-                   pointer_new->xorBpp, pointer_new->colorPtrAttr.width,
-                   pointer_new->colorPtrAttr.height, index));
+        LOG_DEVEL(LOG_LEVEL_WARNING, "lfreerdp_pointer_new: error bpp %d width %d height %d index: %d",
```

Comment:
Can we make this a LOG() rather than a LOG_DEVEL()? See #716 which is still open. If this was a LOG() we might have more useful reports on this by now.

Reply:
done